app/src/Components/Project/ProjectScope.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file displays the project scope document
+ */
+
+import React, { FC } from 'react'
+import { Route, View } from 'react-native'
+import { useTheme } from '@/Hooks'
+import { WebView } from 'react-native-webview'
+
+interface ProjectScopeProps {
+  route: Route
+}
+
+/**
+ * Displays the project scope if a valid url is passed in
+ * @param {Route} route object containing url
+ * @returns {WebView} view of the scope pdf in url
```

Comment:
Looking at lines 22-26 below, I think the return type here should be `ReactElement`.  `WebView` is just one of the things inside the JSX React element (on line 24).